### ISSUE TITLE
VBProject not found message when the workbook does have a VBProject.

### ISSUE THREAD

**Issue by SigleyL25** [ericaddison/ShibbyGit#3]
![shibbygit_error3](https://user-images.githubusercontent.com/20485791/42521911-c0ed3828-841e-11e8-9c1f-12c8c9123a4c.PNG)

![shibbygit_error3_varpathcomparefailed](https://user-images.githubusercontent.com/20485791/42524096-5d066cac-8424-11e8-9b5a-cacd405f0d9e.PNG)

When you have a network mapped file path the workbook variable stores the mapped drive letter with full path. However, the VBProjects file path returns a server name, share path and full path instead of drive letter with full path. This causes the compare logic to fail always on my mapped network location. 

**failed example:**   
        _Z:\_path\fileName  = _\\serverName\shareName\_path\fileName) 
**discovered fix:**  
        `Right(string, Len(string) - InStrRev(string, "\") =  Right(string, Len(string) - InStrRev(string, "\")`                   
                                          *shows fileName only for both

I tried to fix the issue above but then received another separate error  "Run-time error '5': Invalid procedure call or argument.  At function "_GetBooleanFromDocProperties_"  on line  "_val = docProps.Item(name).value_"

![shibbygit_error4](https://user-images.githubusercontent.com/20485791/42526196-03ca5ecc-842a-11e8-9fa5-f55358825900.PNG)

**Comment by krehberger** [comment]
Do you get the same error when you open your excel file with the UNC path and not the mapped drive letter?

**Comment by SigleyL25** [comment]
No, When I open the Excel file using the UNC path it gets past that code block. So this can be a work around for those who have mapped network drives they are using to open their Excel file from. 

However, the second error is still present.

![shibbygit_error4](https://user-images.githubusercontent.com/20485791/42538298-71223c8e-844c-11e8-8200-4b5b8a97cee4.PNG)

**Comment by krehberger** [comment]
Did you save your ShibbyGit Settings?

![image](https://user-images.githubusercontent.com/7827016/42539978-0cf9445e-849d-11e8-9365-c70cb1f600fd.png)
![image](https://user-images.githubusercontent.com/7827016/42540011-2bdc495c-849d-11e8-823b-10d294e09be4.png)

**Comment by SigleyL25** [comment]
When I click "Git Settings" the error shows.

**Comment by krehberger** [comment]
What entries do you see in your excel file advanced properties?
Sorry I have only a german screen copy.
![image](https://user-images.githubusercontent.com/7827016/42540655-9e4de368-849f-11e8-8804-02701d6155be.png)

**Comment by SigleyL25** [comment]
Not sure where this is located. Can you navigate me to this area?   I am using Excel 2016.

**Comment by krehberger** [comment]
In Excel select the tab File and then Advanced Properties (see hardcopy)
![image](https://user-images.githubusercontent.com/7827016/42541657-91781056-84a3-11e8-8ed2-880ef73276ef.png)

**Comment by SigleyL25** [comment]
![image](https://user-images.githubusercontent.com/20485791/42541944-61458a66-8459-11e8-9f89-276147a90571.png)

The VBA project is not protected, but the worksheet/workbooks are.  Is this the issue?

**Comment by krehberger** [comment]
Yes this can be the reason for your issue.
Unprotect your workbook and then the advanced properties should be selectable!

**Comment by SigleyL25** [comment]
Workbook and worksheet is unlocked and still the same issue.

**Comment by SigleyL25** [comment]
Which tab would you like to see?

**Comment by krehberger** [comment]
And what do you see on „advanced properties“?

**Comment by SigleyL25** [comment]
![image](https://user-images.githubusercontent.com/20485791/42542249-de8a0c6c-845a-11e8-9907-b34a472d428c.png)

**Comment by krehberger** [comment]
Now we see the reason for your issue. There are no values in Custom properties.
Did you save the ShibbyGit Settings when the workbook/Worksheet was unprotected?

**Comment by SigleyL25** [comment]
Maybe I have installed it incorrectly.  What are the steps to create this as an AddIn from the source code?   
**I did the following:**

1. Opened a blank Excel workbook
2. Imported all the forms, classes and modules from source
3. Saved the workbook as a  .xlam within my AddIns location 
4. Imported the XML and saved it to the .xlam file
5. Closed .xlam workbook and opened a fresh Excel project and enabled add-ins (ShibbyGit displays on Ribbon)

Am I missing anything?
